src/med_bench/estimation/mr.py

```diff
@@ -0,0 +1,279 @@
+import numpy as np
```

Comment:
not sure about all the file names with abbreviations, should be named `multiply_robust.py`?, also do we want to group some estimators together? as `MultiplyRobust` and `DoubleMachineLearning`are very close? should be discussed further